GrandCentralBoard/Widgets/Bonus/Source/PeopleWithBonususFetchController.swift

```diff
@@ -126,8 +125,8 @@ final class PeopleWithBonusesFetchController {
             switch result {
             case .Success(let image):
                 completionBlock(.Success(person.copyWithImage(image)))
-            case .Failure(let error):
-                completionBlock(.Failure(error))
+            case .Failure:
+                completionBlock(.Success(person.copyWithImage(UIImage(named: "placeholder")!)))
```

Comment:
What do you think about returning from the method an instance of `Person` instead of `Result<Person>` and removing `.NoAvatarURL` error (assertion instead, or making Person.avatarPath not optional)?